wqflask/wqflask/ctl/gn3_ctl_analysis.py

```diff
@@ -0,0 +1,132 @@
+import requests
+import itertools
+
+from utility import genofile_parser
+from utility.tools import GN3_LOCAL_URL
+from utility.tools import locate
+
+from base.trait import create_trait
+from base.trait import retrieve_sample_data
+from base import data_set
+
+
+def process_significance_data(dataset):
+    col_names = ["trait", "marker", "trait_2", "LOD", "dcor"]
+    dataset_rows = [[] for _ in range(len(dataset["trait"]))]
+    for col in col_names:
+        for (index, col_data) in enumerate(dataset[col]):
+            if col in ["dcor", "LOD"]:
+                dataset_rows[index].append(round(float(col_data), 2))
+            else:
+                dataset_rows[index].append(col_data)
+
+    return {
+        "col_names": col_names,
+        "data_set_rows": dataset_rows
+    }
+
+
+def parse_geno_data(dataset_group_name) -> dict:
+    """
+    Args:
+        dataset_group_name: string name
```

Comment:
**nitpick:**  Args should be more descriptive.  And types captures what you've documented as an arg:

```
def parse_geno_data(dataset_group_name -> str) -> dict:
...
```

I think I'll have to look for resources to point you to show you how to improve documentation.  @arunisaac if you have anything in mind, you could share ;)